src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePicker.Flyout.cs

```diff
@@ -67,6 +84,9 @@ void OnPicked(DatePickerFlyout snd, DatePickedEventArgs evt)
 					DateChanged?.Invoke(this, new DatePickerValueChangedEventArgs(evt.NewDate, evt.OldDate));
 				}
 			}
+
+			DatePickerFlyout CreateManagedDatePickerFlyout()
```

Comment:
Add access modifier please :)
```suggestion
			private DatePickerFlyout CreateManagedDatePickerFlyout()
```